elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -296,8 +299,11 @@ public boolean matchesId(String checkId) {
         }
 
         Iterable<T> list;
-        FilterScope filterScope = new FilterScope(requestScope, loadClass);
-        list = tx.loadObjects(loadClass, filterScope);
+        Optional<FilterExpression> permissionFilter;
+        permissionFilter = requestScope.getPermissionExecutor().getReadPermissionFilter(loadClass);
+
+        list = (Iterable<T>) tx.loadObjects(loadClass, permissionFilter,
```

Comment:
There are two sources for filter expressions - permissions and the request filter. This is passing one (the permission filter) but not the other (the request filter).  This should be fixed.